src/System.Security.Cryptography.Xml/tests/.gitignore

```diff
@@ -0,0 +1 @@
+!*.pfx
```

Comment:
Binary data belongs in another repo.

Or just include it as a hex/base64 string in a const somewhere.